src/test/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectTest.java

```diff
@@ -191,4 +194,36 @@ public DescriptorImpl() {
         }
     }
 
+    @Ignore
+    @Issue("JENKINS-30206")
```

Comment:
Could you put this in `JobPropertyStepTest` please, since I presume this is intended as a follow-up to https://github.com/jenkinsci/workflow-plugin/pull/199?